.github/workflows/build-main.yaml

```diff
@@ -0,0 +1,38 @@
+name: Build latest from main and deploy to staging
+
+on:
+  workflow_run:
+    workflows: [ 'Run Tests']
+    branches: [ 'main' ]
+    types: [ completed ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
+
+    steps:
+      - uses: hathitrust/github_actions/build@v1
+        with:
+          image: ghcr.io/${{ github.repository }}-unstable
+          dockerfile: Dockerfile.prod
+          tag: ${{ github.sha }}
+          push_latest: true
+          registry_token: ${{ github.token }}
+
+  deploy-staging:
```

Comment:
Is there anything in staging right now? Might want to remove

Reply:
OK -- I think it will make sense to bring it back when we have a CRMS web app in k8s, just wasn't sure it made sense for a cron job.